### PR TITLE
Allow admins to view the preferences of other users

### PR DESCRIPTION
**Pull request by benbeauchamp7**
From the admin page, admins can now click on a blue "view user" button to see their qualifications and marked preferences. You may only view these on a peer teacher, the button won't appear for other users.

### CODE REVIEW

**Comment by benbeauchamp7** [comment]
Addresses a task in #97, namely the preference viewer

File: src/Components/Admin/AdminEmployeeDetail.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                      </strong>
+                      <div className={qual.qualified? 'success' : 'fail'}>
+                        {qual.qualified? 'Qualified' : 'Not Qualified'}
+                      </div>
+                    </span>
+                  ))}
+              </div>
+                
+              < contexts.blocks.Provider value={blocksPayload} >
+                < SchedulingWindow renderBlockType={PreferenceBlock} options={{
```
Comment (dantebarbieri) [inline src/Components/Admin/AdminEmployeeDetail.tsx]: I imagine this displays the preference dot color, but I could find no examples of this on the site.

**Review by dantebarbieri — APPROVED** [review]
Seems good, though not sure if the `readOnly` for preferences is a standard way to accomplish this.

**Comment by dantebarbieri** [comment]
Before you merge, can you add a display of the Preferred Number of Hours per Person?

File: src/Components/Admin/AdminEmployeeDetail.tsx (hunk 1 above)
Comment (benbeauchamp7) [reply src/Components/Admin/AdminEmployeeDetail.tsx]: image

**Pull request merged**